lms/djangoapps/courseware/module_render.py

```diff
@@ -591,7 +591,7 @@ def handle_deprecated_progress_event(block, event):
             )
         else:
             if user_id != user.id:
-                log.warning("{} tried to submit a completion on behalf of {}".format(user, requested_user_id))
+                log.warning("{} tried to submit a completion on behalf of {}".format(user, user.id))
```

Comment:
This should be user_id, not user.id.  The point is that the user was trying to submit someone else's completion.  Returning their own ID doesn't convey that message.